chisel-wrapper

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+# USAGE:
+#   ./<script-name> [ chisel-cut-OPTIONS ] <slices..>
+# Installs the slices, using
+#   chisel cut "$@"
+# Generates a status file similar to /var/lib/dpkg/status at the CHISEL_DPKG_STATUS_FILE path
+# env vars:
+#   CHISEL_BIN                (optional) the path of chisel binary
+#   CHISEL_CACHE_DIR          (optional) the path which chisel will use to store it's cache
+#   CHISEL_DPKG_STATUS_FILE   (required) the path where the dpkg status file should be generated
+
+set -e
+
+CHISEL_BIN="${CHISEL_BIN:-"./chisel"}"
+CHISEL_CACHE_DIR="${CHISEL_CACHE_DIR:-".chisel-cache"}"
+CHISEL_DPKG_STATUS_FILE="${CHISEL_DPKG_STATUS_FILE}"
+
+_print_error() {
+	echo "Error: $@" >> /dev/stderr
+}
+
+_cleanup_cache() {
```

Comment:
I think you could simplify this script if you simply don't allow `CHISEL_CACHE_DIR` to be defined by the user. 

Does it make sense to run this script twice on the same cache? I doubt it, but still, it could (eg starting from no cache, and running chisel twice on the same filesystem).

What you don't want to see happen is this script deleting something important, or the user's cache by mistake. So why not:

```bash
ephemeral_cache=`mktemp`
CHISEL_CACHE_DIR="${CHISEL_CACHE_DIR:-$ephermeral_cache}"
```

and that's it. No need to clean anything cause unless the users want it, and you'll always have a fresh new directory to serve as chisel cache.

Reply:
Thanks! Added your suggestion.